api/V1/Event.js

```diff
@@ -111,49 +112,49 @@ module.exports = function(app, baseUrl) {
   );
 
   /**
-   * @api {get} /api/v1/events Query events recorded
+   * @api {get} /api/v1/events Query recorded events
    * @apiName QueryEvents
    * @apiGroup Events
    *
    * @apiUse V1UserAuthorizationHeader
+   * @apiParam {Datetime} startTime Return only events after this time (optional)
```

Comment:
You can make them optional with apidoc. I think you just need to put the field name in square brackets. So
 `@apiParam {Datetime} [startTime] Return only events after this time`
http://apidocjs.com/#param-api

Reply:
Ah! Nice. I will fix.